include/libtorrent/http_tracker_connection.hpp

```diff
@@ -84,7 +84,6 @@ namespace libtorrent {
 
 		virtual void on_timeout(error_code const&) {}
 
-		tracker_manager& m_man;
```

Comment:
the base class has this reference too, right? that's why this isn't necessary, is it?